mlos_bench/mlos_bench/environments/base_environment.py

```diff
@@ -224,7 +224,10 @@ def setup(self, tunables: TunableGroups, global_config: Optional[dict] = None) -
         """
         _LOG.info("Setup %s :: %s", self, tunables)
         assert isinstance(tunables, TunableGroups)
-
+        # Assign new values to the environment's tunable parameters:
+        groups = list(self._tunable_params.get_covariant_group_names())
+        self._tunable_params.assign(tunables.get_param_values(groups))
+        # Combine tunables, const_args, and global config into `self._params`:
         self._params = self._combine_tunables(tunables)
```

Comment:
Doesn't `_combine_tunables` already do this, or am I missing something?

Reply:
Nm, I think I see the issue.

Reply:
no, `_combine_tunables()` does not mutate the environment, and I wanted to have that mutation logic explicit, so I put it nere